src/apps/dictionaries/redux/actions.ts

```diff
@@ -354,10 +355,10 @@ export const editDictionaryVersionAction = createActionThunk(
 );
 
 export const recursivelyAddConceptsToDictionaryAction = (
-  sourceUrl: string,
   dictionaryUrl: string,
   rawConcepts: (APIConcept | string)[],
-  bulk: boolean = false
+  bulk: boolean = false,
+  sourceUrl?: string
```

Comment:
What happens if `sourceUrl` is blank but one or more `rawConcepts` is a string? We should raise an error in that case.